src/components/Tutorial/TutorialGestureDiagram.tsx

```diff
@@ -0,0 +1,27 @@
+import { createPortal } from 'react-dom'
+import { css } from '../../../styled-system/css'
+import Command from '../../@types/Command'
+import GesturePath from '../../@types/GesturePath'
+import GestureDiagram from '../GestureDiagram'
+
+/** A portal for conditionally rendering Gesture hints. Used to show an appropriate gesture  below the tutorial UI up top, thus Portal. */
+const TutorialGestureDiagram = ({ gesture }: { gesture: Command['gesture'] }) => {
+  const target = document.getElementById('tutorial-gesture-diagram-portal')
```

Comment:
One thing we'll want to do is put this ID in a shared constant. This will help prevent one changing without the other.